specification/automation/resource-manager/Microsoft.Automation/stable/2021-06-22/account.json

```diff
@@ -0,0 +1,954 @@
+        },
+        "limit": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Gets or sets max limit. -1 for unlimited"
+        },
+        "throttleStatus": {
+          "type": "string",
+          "description": "Gets or sets the throttle status."
+        }
+      },
+      "description": "Definition of Usage."
+    },
+    "UsageCounterName": {
+      "properties": {
+        "value": {
+          "type": "string",
+          "description": "Gets or sets the usage counter name."
+        },
+        "localizedValue": {
+          "type": "string",
+          "description": "Gets or sets the localized usage counter name."
+        }
+      },
+      "description": "Definition of usage counter name."
+    },
+    "UsageListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Usage"
+          },
+          "description": "Gets or sets usage."
+        }
+      },
+      "description": "The response model for the get usage operation."
+    },
+    "Key": {
+      "properties": {
+        "KeyName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Automation key name.",
+          "enum": [
+            "Primary",
+            "Secondary"
+          ],
+          "x-ms-enum": {
+            "name": "AutomationKeyName",
+            "modelAsString": true
+          }
+        },
+        "Permissions": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Automation key permissions.",
+          "enum": [
+            "Read",
+            "Full"
+          ],
+          "x-ms-enum": {
+            "name": "AutomationKeyPermissions",
+            "modelAsString": true
+          }
+        },
+        "Value": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Value of the Automation Key used for registration."
+        }
+      },
+      "description": "Automation key which is used to register a DSC Node"
+    },
+    "KeyListResult": {
+      "properties": {
+        "keys": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Key"
+          },
+          "description": "Lists the automation keys."
+        }
+      }
+    },
+    "AutomationAccountUpdateProperties": {
+      "properties": {
+        "sku": {
+          "$ref": "#/definitions/Sku",
+          "description": "Gets or sets account SKU."
+        },
+        "encryption": {
+          "$ref": "#/definitions/EncryptionProperties",
+          "description": "Set the encryption properties for the automation account"
+        },
+        "publicNetworkAccess": {
+          "type": "boolean",
+          "description": "Indicates whether traffic on the non-ARM endpoint (Webhook/Agent) is allowed from the public internet"
+        },
+        "disableLocalAuth": {
```

Comment:
Enum should be preferred over boolean properties. Consider making this property name localAuth with values Enabled/Disabled.
Refer best practice 'Replace boolean/switch properties with better enum' in https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html

Reply:
All other microservices are developed expecting this value to be boolean and it is expected a boolean for policy

Reply:
Given the API already implemented and you have to meet Vanguard deadline, I am granting exception on this. 

For new properties that will be added in future, implement as enum instead of boolean.